src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -68,9 +80,16 @@ const ProfileSelector = ({ input }) => {
       skip: !profile,
     }
   );
+
   useEffect(() => {
-    if (data) {
-      setProfileName(data?.openscap?.profile_name);
+    if (
+      data &&
+      data.openscap &&
+      typeof data.openscap.profile_name === 'string'
+    ) {
+      setProfileName(data.openscap.profile_name);
+    } else {
+      setProfileName('None');
```

Comment:
I don't think setting the profile name to litteral string `None` is necessary to convert to typescript? Why choosing this particular value?
There were no else case in the `js` version, why adding one was necessary here?

Reply:
The main problem is that it is changing the logic of the component. Which could be okay in another context. Right now the goal is to transform a js to a ts. It's better for trackability if we don't add other more than necessary changes during that step.

Reply:
agree